src/helm/benchmark/scenarios/cleva_scenario.py

```diff
@@ -519,6 +519,12 @@ def process_dialogue_instance(self, row: Dict[str, Any], split: str) -> List[Ins
 class CLEVASubjectKnowledgeScenario(CLEVAScenario):
     """
     The subject knowledge task of CLEVA benchmark.
+    We follow HELM (https://github.com/stanford-crfm/helm/tree/main/scripts/fact_completion) to construct the dataset.
```

Comment:
We follow https://github.com/stanford-crfm/helm/tree/main/scripts/fact_completion to construct the Chinese dataset.